packages/deployment-service/src/marketplace/marketplace.provider.ts

```diff
@@ -61,24 +63,17 @@ export class MarketplaceProvider {
    * @param appId The App Id to make the revision for
    * @param domain The domain to be added to the app for login redirect/signout
    */
-  async updateAppUrls(
-    appId: string,
-    domain: string,
-    developerId: string,
-    name: string,
-    redirectUris?: string[],
-    signoutUris?: string[],
-  ): Promise<void> {
+  async updateAppUrls(appId: string, domain: string): Promise<void> {
     const { access_token, token_type } = await this.authenticate()
+    const appDetails = await this.getAppDetails(appId, token_type, access_token)
 
     await firstValueFrom(
       this.httpService.post<Marketplace.AppRevisionModel>(
         `${this.config.url}/apps/${appId}/revisions`,
         {
-          redirectUris: [...(redirectUris || []), `https://${domain}/login`],
-          signoutUris: [...(signoutUris || []), `https://${domain}/login`],
-          developerId,
-          name,
+          ...appDetails,
```

Comment:
Did this work OK? The model is slightly different so I'd expect you to potentially need to do a bit of data transformation